apps/server/src/migrations/mikro-orm/Migration20240604131554.ts

```diff
@@ -0,0 +1,44 @@
+import { Migration } from '@mikro-orm/migrations-mongodb';
+import * as process from 'node:process';
+
+export class Migration20240604131554 extends Migration {
+	async up(): Promise<void> {
+		// eslint-disable-next-line no-process-env
+		if (process.env.SC_THEME === 'n21') {
+			await this.driver.nativeInsert('instance-configs', {
+				name: 'nbc',
+			});
+			console.info('Instance config was added for nbc');
+		}
+
+		// eslint-disable-next-line no-process-env
+		if (process.env.SC_THEME === 'brb') {
+			await this.driver.nativeInsert('instance-configs', {
+				name: 'brb',
+			});
+			console.info('Instance config was added for brb');
+		}
+
+		// eslint-disable-next-line no-process-env
+		if (process.env.SC_THEME === 'thr') {
+			await this.driver.nativeInsert('instance-configs', {
+				name: 'thr',
+			});
+			console.info('Instance config was added for thr');
+		}
+
+		// eslint-disable-next-line no-process-env
+		if (process.env.SC_THEME === 'default') {
+			await this.driver.nativeInsert('instance-configs', {
+				name: 'default',
+			});
+			console.info('Instance config was added for default');
+		}
+	}
```

Comment:
We have Instacne's where the SC_THEME is set to default but the instance is not the dbc for example audit.
Fun fact there is now where set the name audit for it.
https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/host_vars/ref-audit/cfg.yml

Reply:
Is there a better env to insert a document for each instance? 
Would it be a problem when audit uses default?

Reply:
Okay, little bit more security then :)

Reply:
mh not really and the thing is the `SC_THEME` and other configs like this are in not all configmaps for all server services.

For example the `preview-generator`  know only the envs from the `preview-generator-configmap` and `preview-generator-secret` the don't know anny thing form the `api-configmap` so the `preview-generator` don't know the `SC_THEME` as config

Reply:
You're right, but in our case we just need them in the "normal" server to introduce the collection for each instance. 
Is there anything else otherwise this can be marked as solved, right?

Reply:
I dont see a problem here either.. Is it okay for you @mamutmk5 ?

Reply:
Yes it's okay, have forgotten the context that it is an migration and think about it if it is possible to archive it with out an entry at db level and how it is possible that not all services who need it need access to this db thing.

Reply:
This can be reached with an api which we want to create in another ticket to get the instance id for saving tool previews as superhero